### PR TITLE
add notes for n23897

### PR DESCRIPTION
**Pull request by stickies-v**
cc @hebasto - feel free to have a look and/or leave any thoughts on these notes/questions, but no pressure!

### CODE REVIEW

File: _posts/2022-11-09-#23897.md [hunk 1]
```diff
@@ -7,17 +7,37 @@ authors: [hebasto]
 components: ["refactoring", "validation", "mempool"]
 host: stickies-v
 status: upcoming
-commit:
+commit: ed2d714bd1130291a2f781e644f787d421cdf26e
 ---
 
 _Notes and questions to follow soon!_
```
Comment (LarryRuane) [inline _posts/2022-11-09-#23897.md:13]: remove this line

**Review by LarryRuane — APPROVED** [review]
This will be a difficult one, at least for me! As part of reviewing this, I thought I should review the PR, and answer the questions here. But I still have a ways to go, this stuff is pretty dense (information intensive).

File: _posts/2022-11-09-#23897.md [hunk 2]
```diff
@@ -7,17 +7,37 @@ authors: [hebasto]
...
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- [BIP68](https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki) introduced relative timelocks. They are relative to the block in which the transaction's input(s) were mined, and can be expressed as block height or time difference.
```
[the remark this answers is not in the document]
Comment (stickies-v) [reply _posts/2022-11-09-#23897.md:17]: You're right, that is confusing. Rephrased to
> They are relative to the block(s) in which this transaction's parent transaction(s) were mined. The timelock can be expressed as block height or time difference.

**Comment by stickies-v** [comment]
Thanks a lot @LarryRuane, I've incorporated all your suggestions!

> This will be a difficult one, at least for me! As part of reviewing this, I thought I should review the PR, and answer the questions here. 

If you have any further suggestions after reviewing the PR please let me know so I can update the notes/questions, but it's not strictly necessary or expected (albeit very useful) for these reviews. Having a fresh perspective on whether the questions make sense etc is already really helpful, so thanks again.

**Review by hebasto — COMMENTED** [review]
Post-merge ACK f86c60b1ff77fb10f35956d2ee11b34cb665d809.

**Pull request merged**